torch/csrc/jit/backends/backend_detail.cpp

```diff
@@ -48,6 +50,38 @@ c10::FunctionSchema getExecuteSchema() {
       /*arguments=*/{self, handle, input},
       /*returns=*/{output});
 }
+
+std::unordered_map<std::string, BackendPreprocessFunction>&
+backendPreprocessFunctions() {
+  static std::unordered_map<std::string, BackendPreprocessFunction>
+      preprocess_functions;
+  return preprocess_functions;
+}
```

Comment:
```suggestion
namespace {
std::unordered_map<std::string, BackendPreprocessFunction>&
backendPreprocessFunctions() {
  static std::unordered_map<std::string, BackendPreprocessFunction>
      preprocess_functions;
  return preprocess_functions;
}
}
```

Reply:
To make sure this is visible only in this translation unit and can only be accessed through the methods below.